pytest_timeout.py

```diff
@@ -127,11 +147,13 @@ def pytest_report_header(config):
     """Add timeout config to pytest header."""
     if config._env_timeout:
         return [
-            "timeout: %ss\ntimeout method: %s\ntimeout func_only: %s"
+            "timeout: %ss\ntimeout method: %s\ntimeout func_only: %s\n"
+            "timeout signal_timer: %s"
```

Comment:
This is starting to take up a lot of space in the header.  Maybe we should just make it a single line instead?  Means also we don't have to keep repeating `timeout`.
```suggestion
            "timeout: %ss   method: %s   func_only: %s   signal_timer: %s"
```